source/elements/oneMath/source/domains/rng/device_api/device-distributions.rst

```diff
@@ -78,6 +78,10 @@ basic random number generators.
      -     integer
      -     integer
      -     Bernoulli distribution
+   * -     :ref:`onemath_device_rng_geometric`
+     -     integer
+     -     float
```

Comment:
double-checking: is it `float` for both `int32_t`/`int64_t` or should it be `float`/`double` ?

Reply:
You are correct, `int64_t` mapped to `double` in terms of type.
Here, as an idea, `float` refers to general floating-point data type, not exact `float` type.
The same is done for discrete uniform distr and `integer` type in general in this table.

Please share if you think it make sense to reformat this table

Reply:
hmmm, in the table with Continuous distributions it states float/double separately. Probably it makes sense to change `float` to `floating point` and add a note that it can vary for distributions implementation depending on the integer data type size

Reply:
Applied. @aelizaro please take a look one more time :)